### DISCUSSION TITLE
Uninstallation Issue During Rollback Process

### DISCUSSION THREAD

**Discussion by vijayasekar-duraisamy-agilysys** [wixtoolset/issues#8607] [Questions]
Hi Team,

We are utilizing the Wix Toolset v3.14 to create a Windows executable installer. Within the Bundle.wxs file, under the Bootstrapper section, we have two ExePackage processes that manage the installation and uninstallation of our supporting tools(.exe).

Our expectation is that if any issue occurs during the main installation, the installation of these supporting tools should be rolled back. However, we are facing a issue where only one of the supporting tools is getting uninstalled during the rollback, while the other one remains installed.

We need both supporting tools to be uninstalled if a rollback occurs at any point. Could you please assist us in resolving this issue and let us know if any other details are required?
 
Thank you.

**Comment by bevanweiss** [comment]
You've posted in the wrong discussion topic.
You should close this one, create a new discussion in the correct topic, and then post your bundle.wxs alongside your question.

Hopefully you've already read through this section (which references v3)
https://wixtoolset.org/docs/v3/bundle/
and then the two 'updates'
https://wixtoolset.org/docs/fourthree/
https://wixtoolset.org/docs/fivefour/
